noxfile.py

```diff
@@ -224,6 +232,63 @@ def typing(session: nox.Session):
     )
 
 
+@nox.session
+@nox.parametrize(
+    ["major", "version", "extra_args"],
+    [
+        ["8", "8.0.0a3", []],
+        ["7", "7.5.0", []],
+        [
+            "8",
+            "8.0.0a3",
+            [
+                "--package-dir=tests/test_data/package-files/force_setup_cfg",
+                "--force-generate-setup-cfg",
+            ],
+        ],
```

Comment:
Shouldn't we also have 9 in this list? Or is the problem that there is no `.deps` file for a 9.x.y release yet?

Reply:
These tests require the version that's being tested to be available on PyPI.